server.js

```diff
@@ -1,35 +1,93 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
-mongoose.Promise = Promise;
+import express from "express"
+import cors from "cors"
+import mongoose from "mongoose"
+import listEndpoints from "express-list-endpoints"
+
+import netflixData from "./data/netflix-titles.json"
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.Promise = Promise
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+
+
+const NetflixTitle = mongoose.model('NetflixTitle', {
+  show_id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  date_added: String,
+  release_year: String,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String
+})
+
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    await NetflixTitle.deleteMany()
+    netflixData.forEach((data) => {
+      const newNetflixTitle = new NetflixTitle(data)
+      newNetflixTitle.save()
+    })
+  }
+  seedDatabase()
+}
+
+const port = process.env.PORT || 8080
+const app = express()
+
+app.use(cors())
+app.use(express.json())
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+app.get('/', (req, res) => {
```

Comment:
Maybe it can be added some information about the endpoints here, like a landing page? For example that /netflixtitles gives the whole array of movies and shows? It can also be added to the readme :D